src/main/resources/templates/login_page.html

```diff
@@ -22,10 +22,9 @@
 <body>
 <main>
     <div class="mb-4 text-center">
-        <div class="text-success" th:if="${param.logout}">You have been logged out.</div>
-    </div>
-    <div class="mb-4 text-center">
-        <div class="text-warning" th:if="${param.userExists}">This email already in use. Please login.</div>
+        <div class="text-success" th:if="${param.logout}">You have been LOGGED OUT.</div>
+        <div class="text-warning" th:if="${param.emailExists}">This EMAIL already in use. Please login.</div>
+        <div class="text-warning" th:if="${param.usernameExists}">This USERNAME already in use. Please login.</div>
```

Comment:
If username is already in use, then this doesn't mean that the user can login

Reply:
Yeah, you're right.
You think it should be text-warning message, but on registration_page? Or it should be invalid-feedback error message?

Reply:
You can either one